shims/common/src/main/java/org/apache/hadoop/hive/thrift/HadoopThriftAuthBridge.java

```diff
@@ -286,8 +289,13 @@ static String encodeIdentifier(byte[] identifier) {
       HIVESERVER2, METASTORE
     };
 
+    public static final String HIVE_SERVER2_KERBEROS_CUSTOM_AUTH_CLASS =
```

Comment:
Can you see any way to re-use the variable as defined in the HiveConf class, rather then having to redefine the string itself here?